assets/src/apps/authoring/components/ComponentToolbar/UndoRedoToolbar.tsx

```diff
@@ -22,8 +22,31 @@ const UndoRedoToolbar: React.FC = () => {
     dispatch(redo(null));
   };
 
-  useKeyDown(handleUndo, ['KeyZ'], { ctrlKey: true });
-  useKeyDown(handleRedo, ['KeyY'], { ctrlKey: true });
+  useKeyDown(
+    (ctrlKey, metaKey, shiftKey) => {
+      if (ctrlKey && !shiftKey) {
+        handleUndo();
+      }
+    },
+    ['KeyZ'],
+    { ctrlKey: true },
+  );
+  useKeyDown(
+    (ctrlKey, metaKey) => {
+      if (ctrlKey && !metaKey) handleRedo();
+    },
```

Comment:
changes for request -> On Mac, cmd+y works but it also opens chrome history, so it makes it unusable, can we remove this keyboard control for Mac?